src/test/resources/liquibase/harness/change/expectedSql/cloudspanner/addCheckConstraint.sql

```diff
@@ -0,0 +1 @@
+INVALID TEST -- CloudSpanner doesn't support check constraints
```

Comment:
Note for a potential follow-up pull request: Spanner actually supports this now. In addition, the following features are supported that are listed as not supported in this pull request:
1. Check constraints
2. Default values
3. Sequences (bit-reversed sequences)
4. Modifying the data type of a column (albeit very limited, you can for example change a STRING column to a BYTES column and vice-versa. See https://cloud.google.com/spanner/docs/schema-updates#supported-updates)
5.